### PR TITLE
Mark Do's Autonomy Bootcamp

### PR DESCRIPTION
**Pull request by hepromark**
This is the bootcamp for the CIFAR10 dataset. I have included the training/validation loss vs. epoch, training/validation accuracy vs. epoch, training logs, and some inference results.

### CODE REVIEW

**Review by mgupta27 — APPROVED** [review]
Hey Mark. Looks Good!

I like how you included some pictures of the model classifying the images. One thing I would suggest is to put the blocks in the model into functions so that you don't have to write repetitive code :)
